### PR TITLE
PV-133 PV-131 PV-161 Feature/add DRF

### PR DESCRIPTION
**Pull request by sam-hosseini**
Some comments 😊:

* This PR is done in parallel with [this PR](https://github.com/City-of-Helsinki/parking-permits/pull/18) and they should be reviewed together.
* I recommend reviewing commit by commit.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ if [[ "$WAIT_FOR_OPEN_CITY_PROFILE_GRAPHQL_API" = "true" ]]; then
 fi
 
 if [[ "$WAIT_FOR_PARKING_PERMITS_GRAPHQL_API" = "true" ]]; then
-    ADDRESS=$(echo $PARKING_PERMITS_GRAPHQL_API | cut -c 8-)
+    ADDRESS=$(echo $PARKING_PERMITS_GRAPHQL_API | cut -c 8-31)
```
Comment (mhieta) [inline docker-entrypoint.sh]: @sam-hosseini What is the meaning for this added `31` here?

**Review by mhieta — COMMENTED** [review]
In overall no issues, but can you please explain the meaning for this added `31` here?

File: docker-entrypoint.sh (hunk 1 above)
Comment (sam-hosseini) [reply docker-entrypoint.sh]: Updated based on our discussions in c18722e 👍🏼

Comment (mhieta) [reply docker-entrypoint.sh]: 👍

**Review by mhieta — APPROVED** [review]
Looks good now 👍

**Comment by amanpdyadav** [comment]
Looks good to me. Thanks @sam-hosseini 👍

**Pull request merged**